### ISSUE TITLE
update bower registry

### ISSUE THREAD

**Issue by tomkralidis** [OSGeo/ows.js#7]
@juanmav http://bower.io/search/?q=ows.js any chance we can point this to https://github.com/OSGeo/ows.js?  I'm guessing pushing your local changes / new features upstream at some point.

**Comment by juanmav** [comment]
Yes, it can be done. With this new point we need to add some [tags](https://github.com/juanmav/ows.js/tags) (we are using them). 

In a few day I will do a pull request and then I will change the bower config. is it ok?

**Comment by tomkralidis** [comment]
@juanmav sounds good thanks!
